app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1572,98 +1564,154 @@ function DetailCells({ parent }) {
-              </span>
-              <span className="table-arrow hidden">
-                <i
-                  className="fa fa-angle-down"
-                  onClick={parent.collapseTable}
-                />
-              </span>
-              Taxonomy
-            </th>
-            {parent.render_column_header(
-              "Score",
-              `NT_aggregatescore`,
-              "Aggregate score: ( |genus.NT.Z| * species.NT.Z * species.NT.rPM ) + ( |genus.NR.Z| * species.NR.Z * species.NR.rPM )"
-            )}
-            {parent.render_column_header(
-              "Z",
-              `${parent.state.countType}_zscore`,
-              `Z-score relative to background model for alignments to NCBI NT/NR`
-            )}
-            {parent.render_column_header(
-              "rPM",
-              `${parent.state.countType}_rpm`,
-              `Number of reads aligning to the taxon in the NCBI NT/NR database per million total input reads`
-            )}
-            {parent.render_column_header(
-              "r",
-              `${parent.state.countType}_r`,
-              `Number of reads aligning to the taxon in the NCBI NT/NR database`
-            )}
-            {parent.render_column_header(
-              "%id",
-              `${parent.state.countType}_percentidentity`,
-              `Average percent-identity of alignments to NCBI NT/NR`
-            )}
-            {parent.render_column_header(
-              "log(1/E)",
-              `${parent.state.countType}_neglogevalue`,
-              `Average log-10-transformed expect value for alignments to NCBI NT/NR`
-            )}
-            {parent.render_column_header(
-              "%conc",
-              `${parent.state.countType}_percentconcordant`,
-              `Percentage of aligned reads belonging to a concordantly mappped pair (NCBI NT/NR)`,
-              parent.showConcordance
-            )}
-            <th>
-              <Tipsy content="Switch count type" placement="top">
-                <div className="sort-controls center left">
-                  <div
-                    className={
-                      parent.state.countType === "NT"
-                        ? "active column-switcher"
-                        : "column-switcher"
-                    }
-                    onClick={() => {
-                      parent.setState({ countType: "NT" });
-                    }}
-                  >
-                    NT
-                  </div>
-                  <div
-                    className={
-                      parent.state.countType === "NR"
-                        ? "active column-switcher"
-                        : "column-switcher"
-                    }
-                    onClick={() => {
-                      parent.setState({ countType: "NR" });
-                    }}
-                  >
-                    NR
+class ReportTableHeader extends React.Component {
```

Comment:
Can you move this new component to a separate file `views/report`? Maybe we can move `PipelineSampleReport there too to start breaking it apart.

Or if there is to much work for that or circular dependencies we can do it later.

Reply:
Maybe we should update the name to be more accurate, as this includes the whole table.

Reply:
Updated the name. Will move to separate file in follow-up PR.